### PR TITLE
update pipeline image

### PR DESCRIPTION
**Pull request by sallyom**
* update pipeline components base image to `ilab:1.2` image
* add pull_policy & pull_secret to pipeline

### CODE REVIEW

**Comment by sallyom** [comment]
[test-run](https://rhods-dashboard-redhat-ods-applications.apps.ocp-beta-test.nerc.mghpcc.org/experiments/ilab/02978325-e127-4bef-9ebe-4f8f2e6c0875/runs/c8f6f615-c37d-4cdf-a357-608fb5814b18)

**Comment by sallyom** [comment]
@MichaelClifford I've updated this to separate the sdg pipeline parameter into a separate PR, #146

File: pipeline.py [hunk 1]
```diff
@@ -18,6 +20,7 @@
 JUDGE_SECRET = "judge-server"
 MOCKED_STAGES = ["sdg", "train", "eval"]
 PIPELINE_FILE_NAME = "pipeline.yaml"
+IMAGE_PULL_SECRET = "redhat-et-ilab-botty-pull-secret"
```
Comment (MichaelClifford) [inline pipeline.py:23]: Do we want to give this a more generic name?

File: pipeline.py [hunk 2]
```diff
@@ -9,6 +9,8 @@
     CreatePVC,
     DeletePVC,
     mount_pvc,
+    set_image_pull_policy,
```
Comment (MichaelClifford) [inline pipeline.py:12]: Is `set_image_pull_policy` every used?

**Comment by MichaelClifford** [comment]
Can you rebase this PR also 😄

File: pipeline.py (hunk 1 above)
Comment (sallyom) [reply pipeline.py:24]: we will in a follow-up

File: pipeline.py (hunk 2 above)
Comment (sallyom) [reply pipeline.py:12]: i left in there so when we do need it all we have to do is uncomment below (for now, while we zero in on what's required in the ilab1.2+ image)

**Review by MichaelClifford — APPROVED** [review]
LGTM!

**Pull request merged**